src/main/java/com/philderbeast/prisonpicks/Pick.java

```diff
@@ -116,9 +116,11 @@ void doBreak(Block block, Map<String, Boolean> enchants, Player player, Material
         }
     }
 
+    protected abstract char[] parseInt(String string);
```

Comment:
Class variables should all be at the top of the class

Reply:
so is it needed? if not remove it.

also there is a built in parseInt method, so it should be using that anyway